fix: UIG-2483 - vl-functional-header - gedrag tabs

ksvlaanderen opened the pull request.

Goldflow commented on libs/components/src/lib/tabs/vl-tabs.component.ts:
```diff
@@ -180,18 +184,14 @@ export class VlTabsComponent extends BaseElementOfType(HTMLElement) {
-                }
-            } else {
-                tab.removeActiveClass();
+        const tab = [...this.__tabList.children].find((tab) => tab.id == newValue);
```
goeie wijziging, maar ik zou in de commit wel in bij details beschrijven van wat naar wat het gedrag is veranderd

Goldflow replied:
is het nuttig nog eventuele cypress test toe te voegen?

ksvlaanderen replied:
Done!

Goldflow reviewed (changes requested):
code runs, testen werken
afhankelijk van je respons kan ik het ook zo al approven

ksvlaanderen committed:
fix: UIG-2483 - vl-functional-header - gedrag tabs

Tabs in functional-header aangepast zodat ze zich op dezelfde manier gedragen als buiten de functional-header.
Cypress tests toegevoegd.

Goldflow commented on apps/storybook-e2e/src/e2e/components/functional-header/vl-functional-header.stories.cy.ts:
```diff
@@ -204,6 +204,29 @@ describe('story vl-functional-header tabs', () => {
+        cy.get('@change').should('have.been.calledOnce');
+    });
+
+    it('should open/close tablist on mobile', () => {
```
nice!

Goldflow reviewed (approved):
great!

<img width="350" alt="image" src="https://github.com/milieuinfo/uigov-web-components/assets/15731348/da7e9591-9d26-4668-a5b9-9530e6d8a6c7">

kspeltix commented:
:tada: This PR is included in version 1.8.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.8.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
